README.md

```diff
@@ -1,5 +1,29 @@
 # Northcoders News API
 
+## View the app [here](https://obscure-lowlands-69895.herokuapp.com/api)
+
+--
+
+## About
+
+This is a RESTful API for articles, comments and users. It includes multiple endpoints with various params and queries. You can view `/api` to get a list of all available endpoints.
+
 ## Using this API
 
-You would need to create `.env.test` and `.env.development` and add the names for the database for each, they would be `nc_news_test` and `nc_news` respectively.
+- First you would need to clone this repository.
+
+- Then you would need to create `.env.test` and `.env.development` and add the names for the database for each, they would be `nc_news_test` and `nc_news` respectively. Then follow these steps:
+
+- run `npm run install`
```

Comment:
you could mention the dependancies that you used, so someone could see at a glance